### DISCUSSION TITLE
Flyway issue when upgrade to the latest version

### DISCUSSION THREAD

**Discussion by hantsy** [micronaut-projects/micronaut-data#2858] [Q&A] [answered]
When upgrade to the latest 4.3.x, I got exceptions like the following in the build log, 

The example project:
https://github.com/hantsy/micronaut-sandbox/tree/master/data-jdbc

The complete build log:
https://github.com/hantsy/micronaut-sandbox/actions/runs/8477012590/job/23227335446

```bash
Caused by: org.flywaydb.core.api.FlywayException at DatabaseTypeRegister.java:10
```

**Comment by radovanradic** [comment, marked as answer]
Adding dependency `runtimeOnly("org.flywaydb:flyway-database-postgresql")` seems to resolve the issue. And I think currently when postgres and flyway features are selected in launcher then the dependency is added.

**Reply by hantsy** [reply]
Thanks, resolved the issue.
